packages/core/src/components/select/select.tsx

```diff
@@ -397,7 +397,7 @@ export class Select {
                   ref={(ref) => (this.inputRef = ref)}
                   onInput={() => this.filterItemsWithTypeahead()}
                 />
-                {this.allowClear &&
+                {this.allowClear === true &&
```

Comment:
Is this really neccessary?

Reply:
Without it, calling the prop on javascript would not work. "allow-clear" would not trigger the clear button to be displayed.

Reply:
Sorry I don't get it. Can you please explain in more detail why this won't work?